rdkit/Chem/SaltRemover.py

```diff
@@ -33,20 +33,61 @@
 
 import os
 import re
+from rdkit.Chem.rdmolfiles import SDMolSupplier
 
 from rdkit import Chem
 from rdkit import RDConfig
 from contextlib import closing
 
+class Format:
+  Smarts, Mol, Smiles = range(3)
+
+class SmartsMolSupplier(object):
```

Comment:
Far away of being a **Python** expert, I would agree on that one. And I was not aware of the specialization.